actions/executions_pause.py

```diff
@@ -0,0 +1,25 @@
+from lib.action import St2BaseAction
+
+__all__ = [
+    'St2ExecutionsPause'
+]
+
+
+def format_result(item):
+    if not item:
+        return None
+
+    return item.to_dict()
+
+
+class St2ExecutionsPause(St2BaseAction):
+    def run(self, ids):
+        result = {}
+        for i in ids:
+            try:
+                res = self.client.liveactions.pause(execution_id=i)
+            except Exception as exc:
+                result[i] = '{}'.format(exc)
+            else:
```

Comment:
Also, if one of these fails (exception) then we should probably stop and fail the action, right now the action will always succeed.

Reply:
yes, it's coming! Realized that after running some real-world scenarios :)

Reply:
but not sure if we need to stop? I thought of returning overall result True/False and a dict of results. Perhaps, we can add a flag to ignore/stop on errors as action's parameter? Maybe in a future release if such feature is requested?